printer_interface.py

```
@@ -1,3 +1,676 @@
+            self.paginaavista =  self.canvasfirstpage.create_image((desloc,0), image=self.imagemavista, anchor='n')
+            self.varpageview.set("{} / {} - Pagina: {}".format(str(1), len(self.listadepaginas), paginadoc))
+        except:
+            None
+    def sel_printer(self, name=None, index=None, mode=None, sv=None):
+        if(self.popupMenu2!=None):
+            
+            if plt == "Linux":
+                print(name)
+                comando = "echo kali | lpoptions -d "+self._printer.get() +" -l"
+                #comando = ["lpoptions", "-d", name, "-l"]
+                process = subprocess.Popen(comando, shell=True, stdout=subprocess.PIPE, universal_newlines=True)
```

Ruff (0.15.10)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)